packages/suite/src/components/suite/graph/TransactionsGraph/TransactionsGraph.tsx

```diff
@@ -90,6 +91,76 @@ export interface FiatGraphProps extends CommonProps {
 
 export type TransactionsGraphProps = CryptoGraphProps | FiatGraphProps;
 
+const emptyList: ReturnType<typeof selectAccountTransactionsWithNulls>[] = [];
+const useGraphUpdater = ({
+    shouldUpdateGraph,
+    account,
+}: {
+    shouldUpdateGraph: (abortController: AbortController) => Promise<any> | undefined;
+    account: Account | undefined;
+}) => {
+    const [currentPromise, setCurrentPromise] = useState<{
+        promiseId: string;
+        txIds: string[];
+        promise: Promise<any>;
+        abortController: AbortController;
+    } | null>(null);
+
+    const allTransactions = useSelector(state =>
+        account ? selectAccountTransactionsWithNulls(state, account.key) : emptyList,
+    );
+
+    const newestTransactions = allTransactions
+        .slice(0, 3)
+        .flatMap(tx => tx)
```

Comment:
or what is the purpsoe of flatmap here?
```suggestion
        .flat()
```